<ui version="4.0" >
 <class>pqBarChartDisplayPanel</class>
 <widget class="QWidget" name="pqBarChartDisplayPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QCheckBox" name="ViewData" >
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';">Toggle visibility of this display's series.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="whatsThis" >
      <string>Use this option to show/hide all the line series in the chart.</string>
     </property>
     <property name="text" >
      <string>Show Series in Chart</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Attribute Mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="AttributeMode" >
     <item>
      <property name="text" >
       <string>Points</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqPointData16.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Cells</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqCellData16.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Vertices</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqPointData16.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Edges</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqRamp24.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Rows</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqSpreadsheet16.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="pqTreeWidget" name="CompositeIndex" >
     <column>
      <property name="text" >
       <string>Select Block</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>X Axis Data</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="UseArrayIndex" >
        <property name="whatsThis" >
         <string>Select this option to use the indexes of the y-axis array for the x-axis points.</string>
        </property>
        <property name="text" >
         <string>Use Array Index From Y Axis Data</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="UseDataArray" >
        <property name="whatsThis" >
         <string>Select this option to use a data array to define the x-axis points.</string>
        </property>
        <property name="text" >
         <string>Use Data Array</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="XAxisArray" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="whatsThis" >
         <string>Choose the name of the x-axis data array.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QGroupBox" name="SeriesGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Bar Series</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="pqTreeView" name="SeriesList" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>This displays the list of line series. You can use the list to edit the series options.</string>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers|QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="SeriesEnabled" >
        <property name="whatsThis" >
         <string>Use this option to show/hide the line series in the chart.</string>
        </property>
        <property name="text" >
         <string>Enable Series</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Series Color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="SeriesColorType" >
        <item>
         <property name="text" >
          <string>Solid Color</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Lookup Table</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QStackedWidget" name="ColorButtonStack" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="SolidColorPage" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="pqStandardColorButton" name="ColorActorColor" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Edit the constant color for the geometry.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string>Set Solid Color...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ColorMapPage" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="EditColorMapButton" >
            <property name="text" >
             <string>Edit Color Map...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RescaleButton" >
            <property name="text" >
             <string>Rescale to Data Range</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeView</class>
   <extends>QTreeView</extends>
   <header>pqTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>pqStandardColorButton</class>
   <extends>QToolButton</extends>
   <header>pqStandardColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SeriesColorType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ColorButtonStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>213</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel" >
     <x>147</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseDataArray</sender>
   <signal>toggled(bool)</signal>
   <receiver>XAxisArray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>82</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel" >
     <x>213</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
